packages/core/src/components/views/Banner.tsx

```diff
@@ -72,15 +75,19 @@ export const Banner: React.FC = () => {
       />
       {expanded &&
         bannerMessages.map((message) => (
-          <BannerSection
-            id={message.id}
-            key={message.id}
-            title={message.title}
-            type={message.type}
-            variant="detail"
-            onDismiss={() => dismissBanner(message.id)}
-            dismissible={message.dismissible}
-          />
+          <React.Fragment key={message.id}>
+            {/* Render a divider between the banners when multiple banners exist */}
+            <View style={{ height: 2, backgroundColor: ColorPallet.grayscale.white }} />
```

Comment:
Kind of a nit but can we use brand colors? White might be a bit jarring on dark themes
```suggestion
            <View style={{ height: 2, backgroundColor: ColorPallet.brand.primaryBackground }} />
```

Reply:
Updated!

I also included a small tweak to the banner message dispatch. Just to make sure they are all unique.